src/hotspot/share/opto/addnode.cpp

```diff
@@ -235,6 +235,45 @@ const Type *AddNode::add_of_identity( const Type *t1, const Type *t2 ) const {
   return NULL;
 }
 
+//---------------------------Bitmask helper------------------------------------
+// Get a bitmask for expression: "(a & 0xFF) << 8" -> 0xFF00
```

Comment:
This comment is thoroughly misleading and inadequate. Please describe correctly what this routine does and why.